documents/extension/funding/service.go

```diff
@@ -183,12 +181,17 @@ func (s service) DeriveFromPayload(ctx context.Context, req *clientfundingpb.Fun
 		return nil, documents.ErrDocumentNotFound
 	}
 
+	ca, err := documents.FromClientCollaboratorAccess(req.ReadAccess, req.WriteAccess)
```

Comment:
Maybe my lack of knowledge on this feature, but why do we need to pass collaborators on the funding request level and not on the fundingData level?